RockPaperScissors/RockPaperScissors/UserCards.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  UserCards.swift
+//  RockPaperScissors
+//
+//  Created by 박종화 on 2023/05/03.
+//
+
+enum UserCards: Int {
+    case rock = 1
+    case scissors = 2
+    case paper = 3
+    case exit = 0
+}
```

Comment:
enum 타입 활용 👍

참고로, enum 타입의 rawValue Type을 Int로 주신 경우 case에 기본값을 할당하지 않으면 가장 위 case를 0으로 시작해 1씩 증가하게 됩니다.
```suggestion
enum UserCards: Int {
    case exit
    case rock
    case scissors
    case paper
}
```

Reply:
오.. 몰랐던 사실이군요.. 앞으로도 enum 사용할 때 0으로 시작하는 값을 할당하게 된다면 한 번 사용해보겠습니다!!! 감사합니다!!!